index.html

```diff
@@ -47,6 +47,7 @@
     <script type="text/javascript" src="js/plugins.js"></script>
     <script type="text/javascript" src="js/imgur.js"></script>
     <script type="text/javascript" src="3rdparty/favico-0.3.10.min.js"></script>
+    <script type="text/javascript" src="cordova.js"></script>
```

Comment:
this is something that'll really be better off in a cordova branch (or maybe added during the cordova build process), otherwise it always creates failed requests